packages/launcher/lib/detect.ts

```diff
@@ -24,6 +31,10 @@ const setMajorVersion = (browser: FoundBrowser) => {
       browser.version,
       browser.majorVersion
     )
+
+    if (browser.majorVersion) {
+      browser.majorVersion = parseInt(browser.majorVersion)
```

Comment:
you pretty much always want `parseFloat` here or `Number(...)`. `parseInt` is supposed to always have `radix` although i know in newer browsers it doesn't matter.